### PR TITLE
Bug fixes and Summary feature

### PR DESCRIPTION
**Pull request by Kratika0907**
Bug fix for cart and shop image 
Add summary feature

### CODE REVIEW

File: components/shoplist-card/index.js [hunk 1]
```diff
@@ -1,23 +1,22 @@
...
-    {
-      path.join('./assets', product.image);
-    }
     props.del.delShopListItem(item);
   };
   return (
     <div className="shop-card">
-      <img src={path.join('./assets', product.image)} alt={product.name} />
+      <Link href={props.link.href} as={props.link.as}>
+        <img src={path.join('./assets', product.image)} alt={product.name} />
```
Comment (sumitd94) [inline components/shoplist-card/index.js:18]: could you please re-check the first argument path

Comment (Kratika0907) [reply components/shoplist-card/index.js:18]: yes that is correct, it is wrt to public dir

File: components/product-details/index.js [hunk 2]
```diff
@@ -6,7 +7,7 @@ export const ProductDetails = ({ productName }) => {
     <div className="product-container">
       <img
         className="product-image"
-        src="../../styles/user-female-alt-icon.png"
+        src={path.join('../assets', product.image)}
```
Comment (sumitd94) [inline components/product-details/index.js:10]: I think this needs to be then path.join('./assets',product.image)

Comment (Kratika0907) [reply components/product-details/index.js:10]: No since this one level inside component page

**Pull request merged**